File: 800/236A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string s;
    cin >> s;

    set<char> distinctChars(s.begin(), s.end());

    if (distinctChars.size() % 2 == 0)
        cout << "CHAT WITH HER!";
    else
        cout << "IGNORE HIM!";

    return 0;
}

File: 800/236A/main.java
import java.util.*;

public class main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();

        // Use HashSet to store distinct characters
        HashSet<Character> distinctChars = new HashSet<>();
        for (char c : s.toCharArray()) {
            distinctChars.add(c);
        }

        if (distinctChars.size() % 2 == 0) {
            System.out.println("CHAT WITH HER!");
        } else {
            System.out.println("IGNORE HIM!");
        }
    }
}